rest/route/generate_routes.go

```diff
@@ -0,0 +1,63 @@
+package route
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/gorilla/mux"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+func getGenerateManager(route string, version int) *RouteManager {
+	h := &generateHandler{}
+	generate := MethodHandler{
+		RequestHandler: h.Handler(),
+		MethodType:     http.MethodPost,
+	}
+
+	generateRoute := RouteManager{
```

Comment:
nit: just return the RouteManager instead of naming it.

Reply:
done